sufficient_statistics.cc

```diff
@@ -72,6 +78,8 @@ double SufficientStatistics::GetHomozygousStatistic(const TrioModel &params) {
  */
 double SufficientStatistics::GetMismatchStatistic(const TrioModel &params) {
   return GetSequencingErrorStatistic(params, 2);
+    //Note: Circular dependence: SetReadDependentData(data_vec) in Update().  read_dependent_data() here
+    //Note: GetMismatches(heter/homo) can factor out, it's constant for given dataset
```

Comment:
ignore this one, it wast from the last round
On 02/28/2015 02:20 PM, Melissa Ip wrote:
>
> In sufficient_statistics.cc 
> <https://github.com/maip/novo_muta/pull/1#discussion_r25561333>:
>
> > @@ -72,6 +78,8 @@ double SufficientStatistics::GetHomozygousStatistic(const TrioModel &params) {
> >   */
> >  double SufficientStatistics::GetMismatchStatistic(const TrioModel &params) {
> >    return GetSequencingErrorStatistic(params, 2);
> > +    //Note: Circular dependence: SetReadDependentData(data_vec) in Update().  read_dependent_data() here
> > +    //Note: GetMismatches(heter/homo) can factor out, it's constant for given dataset
>
> I don't understand what you mean. I added the matches as part of the 
> ReadDependentData, so all we pass in now is the params.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/maip/novo_muta/pull/1/files#r25561333>.
>